maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/XJCVersion.java

```diff
@@ -0,0 +1,64 @@
+package org.jvnet.jaxb.maven;
+
+public class XJCVersion {
+    private String raw = "N/A";
```

Comment:
Perhaps add a public static final String NOT_DETECTED = "NOT DETECTED" <-- as a way to signal to users that there is a tooling version mismatch?

Reply:
method isKnown will return true if at least one version (major, minor, bugfix) is > 0